### ISSUE TITLE
Package nothing expect the plugin binary

### ISSUE THREAD

**Issue by ialidzhikov** [aliyun/terraform-provider-alicloud#2781]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Not applicable.

### Expected Behavior

Starting from v1.92.0 terraform-provider-alicloud plugin packages not only the plugin binary, but also the `CHANGELOG.md`, `LICENSE`, `README.md` files.

For `v1.94.0` we have:

```
$ wget https://releases.hashicorp.com/terraform-provider-alicloud/1.94.0/terraform-provider-alicloud_1.94.0_linux_amd64.zip

$ unzip terraform-provider-alicloud_1.94.0_linux_amd64.zip
Archive:  terraform-provider-alicloud_1.94.0_linux_amd64.zip
  inflating: CHANGELOG.md
  inflating: LICENSE
  inflating: README.md
  inflating: terraform-provider-alicloud_v1.94.0

$ ls -la
total 229320
drwxr-xr-x  7 me staff       224 Aug 31 19:19 .
drwxr-xr-x 11 me staff       352 Aug 31 19:19 ..
-rw-r--r--  1 me staff    262092 Aug 24 07:09 CHANGELOG.md
-rw-r--r--  1 me staff     16725 Aug 12 05:01 LICENSE
-rw-r--r--  1 me staff      4143 Aug 24 07:09 README.md
-rw-r--r--  1 me staff  33540958 Aug 24 10:02 terraform-provider-alicloud_1.94.0_linux_amd64.zip
-rwxr-xr-x  1 me staff 188645376 Aug 24 07:09 terraform-provider-alicloud_v1.94.0
```

For `v1.84.0` version of terraform-provider-alicloud we have the expected behaviour:
```
$ wget https://releases.hashicorp.com/terraform-provider-alicloud/1.84.0/terraform-provider-alicloud_1.84.0_linux_amd64.zip

$ unzip terraform-provider-alicloud_1.84.0_linux_amd64.zip
Archive:  terraform-provider-alicloud_1.84.0_linux_amd64.zip
  inflating: terraform-provider-alicloud_v1.84.0_x4

$ ls -la
total 201248
drwxr-xr-x  4 me staff       128 Aug 31 19:37 .
drwxr-xr-x 11 me staff       352 Aug 31 19:36 ..
-rw-r--r--  1 me staff  30876443 May 22 21:12 terraform-provider-alicloud_1.84.0_linux_amd64.zip
-rwxr-xr-x  1 me staff 172552192 May 22 21:06 terraform-provider-alicloud_v1.84.0_x4
```

aws, azurerm, google, openstack are all packaging only the plugin binary. terraform-provider-alicloud should be consistent with them.

### Actual Behavior
`CHANGELOG.md`, `LICENSE`, `README.md` files are bundled in terraform-provider-alicloud package.

**Comment by xiaozhu36** [comment]
@ialidzhikov Thanks. I will check it.

**Comment by xiaozhu36** [comment]
Fixed by v1.95.0.
